src/Avalonia.Visuals/Animation/Animators/SolidColorBrushAnimator.cs

```diff
@@ -3,37 +3,39 @@
 using Avalonia.Media;
 using Avalonia.Media.Immutable;
 
+#nullable enable
+
 namespace Avalonia.Animation.Animators
 {
     /// <summary>
     /// Animator that handles <see cref="SolidColorBrush"/> values. 
     /// </summary>
-    public class ISolidColorBrushAnimator : Animator<ISolidColorBrush>
+    public class ISolidColorBrushAnimator : Animator<ISolidColorBrush?>
```

Comment:
though i apparently did the same mistake about class name being confused as an Interface